include/mlir/Transforms/DialectConversion.h

```diff
@@ -382,6 +382,9 @@ class ConversionPatternRewriter final : public PatternRewriter {
   /// Return a reference to the internal implementation.
   detail::ConversionPatternRewriterImpl &getImpl();
 
+  /// Remap the given value to that with potentially different type.
```

Comment:
Can you reword this a bit? Maybe mention that this grabs the converted value that replaced 'key'.

Can you also move this to above the `PatternRewriter Hooks` section. This part is intended solely for the implementations of the `PatternRewriter` hooks.